### ISSUE TITLE
Folder hierarchy for records

### ISSUE THREAD

**Issue by Onlyodin** [fcturner/passwords#200]
It would be great if records could be arranged under folders, similar to files within owncloud. I have used KeePass and KeePassX, and this feature seems almost mandatory for anything other than personal use.

**Comment by fcturner** [comment]
I see. I need to think about a way that would work well in terms of user experience, but I like the idea.

**Comment by Vebryn** [comment]
Stay simple, tags are *on my opinion* a better idea. Look at Gmail, it no more uses folders. I think tags are 21 century way to sort.

**Comment by fcturner** [comment]
Yes, maybe that's a better way. It could replace categories as they currently are.

**Comment by Onlyodin** [comment]
Either way could work. Coupled with #207 if there can be a default (customised per user?) filter, it might reduce the number of records that need to be decrypted, therefore improving loading times?
